MethodsAndParameterizedTests/src/WordsAndChars.java

```diff
@@ -0,0 +1,43 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class WordsAndChars {
+
+    boolean areUniqueCharacters(String text) {
+        List<Character> characters = new ArrayList<>();
+        for (int i = 0; i < text.length(); i++) {
+            char character = text.charAt(i);
+            if (characters.contains(character)) {
```

Comment:
Or you can use set and decrease complexity :)

Reply:
to see more: https://stackoverflow.com/questions/559839/big-o-summary-for-java-collections-framework-implementations

the point is when you hit `ArrayList.contains`, your algorithmic complexity will be `O(n)`; if you do the same with a `HashSet`, you will get `O(1)`